src/Resources/contao/drivers/DC_Table.php

```diff
@@ -2172,6 +2181,16 @@ public function edit($intId=null, $ajaxId=null)
 							   ->execute(time(), $this->intId);
 			}
 
+			$arrRecords = $objSessionBag->get('edit_records', array());
+			$intLatestVersion = $objVersions->getLatestVersion();
+
+			// Show a warning if the record has been saved by another user (see #8412)
+			if ($intLatestVersion !== null && isset($arrRecords[$this->strTable][$this->intId]) && $intLatestVersion > ($arrRecords[$this->strTable][$this->intId] + 1))
+			{
+				\Message::addError(sprintf($GLOBALS['TL_LANG']['ERR']['versionWarning'], ($intLatestVersion - 1), $arrRecords[$this->strTable][$this->intId]));
+				$this->reload();
```

Comment:
Ah, I see. So it does actually not prevent you from overriding the values but it just tells you that you did override it. That's fine for me I guess.